src/StatsdClient/project.json

```diff
@@ -0,0 +1,19 @@
+{
+  "version": "1.0.0-*",
+
+  "dependencies": {
+    "NETStandard.Library": "1.5.0-rc2-24027",
+    "NUnit": "3.2.1"
+    
+  },
+
+  "frameworks": {
+    "net451": {},
+    "netstandard1.5": {
```

Comment:
.NetCoreApp is a platform and .NetStandard.Library is a library supposed to be cross platform (portable class library) for various .NET platforms runtimes.

You can include a direct reference (package) of NetStandard.Library in any of your .NET platform project that is supported, for ex .NETCoreApp (Dot Net Core 1.X)